README.md

```diff
@@ -5,10 +5,25 @@ This plugin helps you leverage OpsLevel's powerful maturity features with your e
 ![](docs/rubric.png)
 ![](docs/maturity.jpg)
 
+[OpsLevel](https://www.opslevel.com/) gives platform engineers and DevOps leaders the tools to drive service maturity for their teams. We understand that maturity features alongside your software catalog are critical to a culture of service ownership. Which is why we’ve made our maturity features available to Backstage users within their existing service catalog.
+
+OpsLevel uses [Checks](https://www.opslevel.com/docs/getting-started-with-checks) to measure Service Maturity. Checks let you explicitly define how you want your services to be built and operated.
+With OpsLevel Checks, you can verify that services:
+* Are using a particular version of a library or framework
+* Have migrated to a new third-party tool (e.g., all services use Splunk)
+* Meet certain operational requirements (e.g., all Tier-1 services have an on-call schedule)
+And a whole lot more.
```

Comment:
Need whitespace so that this shows up in a newline and not part of the last bullet.

```suggestion

And a whole lot more.
```